packages/coding-standard/src/Rules/NoProtectedElementInFinalClassRule.php

```diff
@@ -49,9 +49,6 @@ public function getNodeTypes(): array
     public function process(Node $node, Scope $scope): array
     {
         $parent = $node->getAttribute('parent');
-        if (! $parent instanceof Class_) {
-            return [];
-        }
```

Comment:
What about traits now?

Reply:
I checked with trait too in fixture, it always pass.